### ISSUE TITLE
Rest API, fields param not work

### ISSUE THREAD

**Issue by ilylx** [emqx/emqx#12816]
### What happened?

/api/v5/clients?page=1&limit=50&conn_state=connected&fields=clientid

### What did you expect to happen?

Only clientid field return in  the respose.

### EMQX version

<details>

```console
$ ./bin/emqx_ctl broker
# paste output here
```

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here
```

</details>

### Log files

<details>

</details>

**Comment by ieQu1** [comment]
Hello,

Which version of EMQX are you using?

**Comment by zmstone** [comment]
this feature is released in 5.6.0
PR: https://github.com/emqx/emqx/pull/12719

original release note: https://github.com/emqx/emqx/pull/12719/files#diff-74d4255555e60811136e82ed15ace38fe0792726a305e5eedf5875d2e6ebf0c7
